content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -39,6 +39,11 @@ Resource quotas work like this:
   See the [walkthrough](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)
   for an example of how to avoid this problem.
 
+{{< note >}}
+- For cpu/memory: ResourceQuota works only if every other pods in the namespace sets a limit for that resource. It means that you cannot create a new pod without limit/request CPU if the resource quota limits the CPU quota of this namespace.
```

Comment:
Is this better?
```suggestion
For `cpu` and `memory` resources, ResourceQuotas only work if **every**
other pods in the same namespace sets a limit for that resource.
If you enforce a resource quota in a namespace for either `cpu` or `memory`,
you (and other clients) **must** specify either `requests` or `limits` for that resource,
for every new Pod you submit. If you don't, the control plane may reject admisison
for that Pod.
You can use a [LimitRange ](/docs/concepts/policy/limit-range/)to automatically set
a default request for these resources.
```

Reply:
@sftim the suggested notes look ok to me but i want to address this
>If you don't, the control plane may reject admisison
for that Pod.
>
to

`If you don't, the control plane will reject admisison
for that Pod.`

and 
> **must** specify either `requests` or `limits`
>
to
**must** specify either `requests` and `limits`

I am not so sure that if we provide either of this will work, i think we have to explicitly set both the request and limit in pod   manifest , any thoughts on this @pacoxu @YuikoTakada , please suggest.

Reply:
It depends.
```
apiVersion: v1
kind: ResourceQuota
metadata:
  name: compute-resources
spec:
  hard:
    requests.cpu: "1"
    requests.memory: 1Gi
    limits.cpu: "2"
    limits.memory: 2Gi
```
You can special both in resourcequota or just limit or just request for cpu/memory.

Reply:
Sure, i have addressed the changes please take a look